pkg/util/voljournal.go

```diff
@@ -26,6 +28,8 @@ import (
 	"k8s.io/klog"
 )
 
+const uuidEncodedLength = 36
```

Comment:
@ShyamsundarR anything special about the length of `36`. Just trying to understand..

Reply:
This is the string representation of a UUID which is a constant length of 36 bytes ([reference](https://github.com/ceph/ceph-csi/blob/ca8dd2d8f28b6d7747330dc11b77c622ab76070f/vendor/github.com/pborman/uuid/uuid.go#L24-L28))

Unfortunately there is no constant in the uuid package to import and use, hence defined here. This needs a comment though to state it is the UUID string representation length. Will add it in the next revision to the patch set.

Reply:
Addressed in the latest update.

Reply:
Tried the same in go playground, it is disallowed. I could have replaced the single instance of the use with len() as above, but I **do** want to remember the length as this 36, as that was what was used in the past. So if we dynamically use len of the UUID and that changes, assumptions in code will change. Further added a note to this effect.